ln-simln-jamming/src/lib.rs

```diff
@@ -38,12 +42,22 @@ pub fn endorsement_from_records(records: &CustomRecords) -> EndorsementSignal {
     }
 }
 
+/// Converts a set of custom tlv records to a bool signaling if the endorsement signal is
+/// upgradable.
+pub fn upgradable_from_records(records: &CustomRecords) -> bool {
+    match records.get(&UPGRADABLE_TYPE) {
+        Some(upgradable) => upgradable.len() == 1 && upgradable[0] == 1,
+        None => false,
+    }
+}
+
 /// Converts an endorsement signal to custom records using the blip-04 experimental TLV.
 pub fn records_from_endorsement(endorsement: EndorsementSignal) -> CustomRecords {
+    let mut records = CustomRecords::default();
+    records.insert(UPGRADABLE_TYPE, vec![1]);
```

Comment:
nit: update docs to note that this will hard-set the `UPGRADABLE_TYPE`? 

Works for now because we always want this to be set, but maybe in the future we want to have some values that aren't upgradable.